sdk/eventgrid/perf-tests/eventgrid/package.json

```diff
@@ -28,9 +28,9 @@
     "build": "npm run clean && tsc -p .",
     "build:samples": "echo skipped",
     "build:test": "echo skipped",
-    "check-format": "prettier --list-different --config ../../../../.prettierrc.json --ignore-path ../../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "check-format": "prettier --list-different --config ../../../../.prettierrc.json --ignore-path ../../../../.prettierignore \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-* types *.tgz *.log",
-    "format": "prettier --write --config ../../../../.prettierrc.json --ignore-path ../../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "format": "prettier --write --config ../../../../.prettierrc.json --ignore-path ../../../../.prettierignore \"test/**/*.ts\" \"*.{js,json}\"",
```

Comment:
what is up with removing `src` from formatting? is that by mistake?

Reply:
No, there is no src/ folder inside the perf tests. If I don't remove that path from the script, it will throw an error when running `format` or `check-format`. Already talked with @HarshaNalluru about it, there will not be a src/ folder in the future.

Reply:
Thanks for explaining! You're right! I did not notice this is for perf test.